setup.py

```diff
@@ -56,7 +56,6 @@
         "pandas",
         "progressbar2",
         "pyproj>=2.1",
-        "pys2index",
```

Comment:
Why is this removed?

Reply:
because it's not on PyPI

Reply:
That shouldn't make any difference, see https://docs.esmvaltool.org/en/latest/community/code_documentation.html#dependencies